packages/rOpenScPCA/tests/testthat/test-calculate-clusters.R

```diff
@@ -67,6 +67,27 @@ test_that("calculate_clusters runs with additional cluster_args", {
   )
 })
 
+test_that("calculate_clusters runs with an object, defaults", {
+  cluster_df_sce <- calculate_clusters(sce)
+  expect_equal(
+    names(cluster_df_sce),
```

Comment:
I'd probably make this change everywhere you are comparing column names? (row names we do care about order for)
```suggestion
  expect_setequal(
    colnames(cluster_df_sce),
```